src/test/regress/citus_tests/run_test.py

```diff
@@ -26,7 +26,7 @@
 use_base_schedule = args['use_base_schedule']
 use_whole_schedule_line = args['use_whole_schedule_line']
 
-test_files_to_skip = ['multi_cluster_management', 'multi_extension', 'multi_test_helpers']
+test_files_to_skip = ['multi_cluster_management', 'multi_extension', 'multi_test_helpers', 'multi_insert_select']
```

Comment:
`multi_insert_select` is in the same parallel group with `multi_behavioral_analytics_create_table_superuser`, which is also included in the base schedule. We add the test files that are in minimal/base schedules into `test_files_to_skip` as they cannot be ran in the flaky check. The main reason for that is they create some objects that other test files depend on, so we cannot drop those objects at the end of the file, so they cannot be executed 8 times in a row as they throw "object already exists" errors and cause flaky check failure.

We should either add `multi_insert_select` to `test_files_to_skip`, or execute `multi_behavioral_analytics_create_table_superuser` and `multi_insert_select` seperately by removing the parallel group.

Reply:
could use a small comment in the code